src/unix/process.c

```diff
@@ -40,7 +40,7 @@
 extern char **environ;
 #endif
 
-#ifdef __linux__
+#if defined(__linux__) || defined(__GLIBC__)
```

Comment:
I suppose this raises the interesting question if e.g. kfreebsd + musl works / should work.